apis/server/container_bridge.go

```diff
@@ -280,6 +280,12 @@ func (s *Server) getContainer(ctx context.Context, rw http.ResponseWriter, req *
 		State:      meta.State,
 		Config:     meta.Config,
 		HostConfig: meta.HostConfig,
+		GraphDriver: &types.GraphDriverData{
+			Name: "overlay2",
+			Data: map[string]string{
+				"BaseFS": meta.BaseFS,
```

Comment:
for overlay2 it should be `MergedDir` and if you can get the `UpperDir` best put it here too, it is very convenient for user to get these two directories in the inspect result.

Reply:
image is managed by containerd, container have no provided image's mount information, we get   these information just from /proc/mounts. so container must be running.